src/NodaTime/Text/LocalDateTimePattern.cs

```diff
@@ -32,17 +32,22 @@ public sealed class LocalDateTimePattern : IPattern<LocalDateTime>
 
         /// <summary>
         /// Gets an invariant local date/time pattern which is ISO-8601 compatible, down to the second.
-        /// This corresponds to the text pattern "uuuu'-'MM'-'dd'T'HH':'mm':'ss", and is also used as the "sortable"
-        /// standard pattern.
+        /// This corresponds to the text pattern "uuuu'-'MM'-'dd'T'HH':'mm':'ss".
         /// </summary>
+        /// <remarks>
+        /// This pattern corresponds to the 'S' standard pattern ("shorter sortable").
```

Comment:
This should be `the 's' standard pattern`, no?

Reply:
Doh! Yes, absolutely.